linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_commonauth/rule.yml

```
@@ -36,7 +36,7 @@ identifiers:
     cce@sle15: CCE-85754-0
 
 references:
-    disa@sle15: CCI-000803
+    disa: CCI-000803
     nist@sle15: IA-7,IA-7.1
```

This is also product independent. Sorry for not noticing that earlier.
```suggestion
    nist: IA-7,IA-7.1
```